### ISSUE TITLE
About the data augmentation of PASCAL Context dataset?

### ISSUE THREAD

**Issue by yuzhegao** [pkuCactus/BDCN#32]
Hi @pkuCactus 
From your paper, I have knew that using the PASCAL Context dataset as additional training dataset   can help improve the performance( ODS 0.806 -> 0.820)，so I wonder some training detail.  Could your please tell me whether you augment the Pascal Context dataset while training?

**Comment by pkuCactus** [comment]
Yes, we did augment with Pascal Context while training.

**Comment by ShreyasSkandanS** [comment]
@pkuCactus what information did you use for ground truth edges in Pascal Context, the semantic class boundaries? Do you have the data for this?

I'm trying to train your network with the HED-BSDS dataset provided but haven't been able to achieve these accuracy values after 40k iterations.

**Comment by pkuCactus** [comment]
Yes, we indeed use the semantic class boundaries of Pascal Context to augment the training data. Moreover, we also do augmentation on the pascal context to further increase the training data. And then combine the augmented bsds and pascal context for training. However, I have graduated and the data are stored in the school server, I'm not available now. You can download the data from the link and do the augmentation by yourself.

**Comment by ForawardStar** [comment]
@pkuCactus  Where is the link? And how to do augmentation?

**Comment by liuzhidemaomao** [comment]
@pkuCactus
Hello~! BDCN is a nice job! Can you tell me what augmentations do you have applied for Pascal Context?
